discovery-provider/alembic/versions/35198266d709_index_reactions.py

```diff
@@ -0,0 +1,46 @@
+"""index reactions
+
+Revision ID: 35198266d709
+Revises: 378ef6680606
+Create Date: 2022-05-04 19:53:54.350351
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "35198266d709"
+down_revision = "378ef6680606"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "reactions",
+        sa.Column("id", sa.Integer(), nullable=False, autoincrement=True),
+        sa.Column("slot", sa.Integer(), nullable=False),
+        sa.Column("reaction", sa.Integer(), nullable=False),
+        sa.Column("sender_wallet", sa.String(), nullable=False),
+        sa.Column("entity_type", sa.String(), nullable=False),
+        sa.Column("entity_id", sa.String(), nullable=False),
+        sa.Column("timestamp", sa.DateTime(), nullable=False),
+        sa.Column("txid", sa.String(), nullable=True),
```

Comment:
I think we call this `tx_signature` or `signature` elsewhere. We should pick one and stick with it